testData/solutions/win32.json5

```diff
@@ -5807,6 +6472,458 @@
+                            "match": {
+                                "0": "RoutingFocus",
+                                "1": "RoutingHybrid"
+                            },
+                            "noMatch": "RoutingFocus"
+                        }
+                    ]
+                }
+            },
+            "configureMouseCursorShadow": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "getAction": "SPI_GETCURSORSHADOW",
+                    "setAction": "SPI_SETCURSORSHADOW",
+                    "uiParam": 0,
+                    "pvParam": {
+                        "type": "BOOL"
+                    },
+                    "fWinIni": "SPIF_UPDATEINIFILE|SPIF_SENDCHANGE",
+                    "verifySettings": true
+                },
+                "supportedSettings": {
+                    "MouseCursorShadowConfig": {
+                        "schema": {
+                            "title": "Enable disable mouse cursor shadow",
+                            "description": "Changes the way mouse scrolling is passed to applications",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "MouseCursorShadowConfig": {
+                        "transform": {
+                            "type": "fluid.transforms.value",
+                            "inputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.EnableCursorShadow",
+                            "outputPath": "value"
+                        },
+                        "path": {
+                            "transform": {
+                                "type": "fluid.transforms.literalValue",
+                                "input": "pvParam"
+                            }
+                        }
+                    }
+                },
+                "inverseCapabilitiesTransformations": {
+                    "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.EnableCursorShadow":  "MouseCursorShadowConfig.value"
+                }
+            },
+            "configureHorizontalScrollChars": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "getAction": "SPI_GETWHEELSCROLLCHARS",
+                    "setAction": "SPI_SETWHEELSCROLLCHARS",
+                    "uiParam": 0,
+                    "pvParam": {
+                        "type": "UINT"
+                    },
+                    "fWinIni": "SPIF_UPDATEINIFILE|SPIF_SENDCHANGE",
+                    "verifySettings": true
```

Comment:
Remember to globally remove all "verifySettings" (not just the ones you added here)

Reply:
Really nice catch, I can't remember when the pulls intersect, I'm going to remove all the ones I found in the SR from here, and then move to the other pulls.